src/main/java/se/sundsvall/casemanagement/integration/byggr/ByggrMapper.java

```diff
@@ -556,9 +556,30 @@ static ArrayOfHandling createNeighborhoodNotificationArrayOfHandling(final ByggR
 		return new ArrayOfHandling().withHandling(handlingar);
 	}
 
+	static HandelseIntressent createAddAdditionalDocumentsHandelseIntressent(final StakeholderDTO stakeholder, final String stakeholderId) {
+		var handelseIntressent = new HandelseIntressent()
+			.withPersOrgNr(stakeholderId)
+			.withAdress(stakeholder.getAddresses().getFirst().getStreet())
```

Comment:
Nullcheck here? Maybe done before, just double checking.

Reply:
The validation on caseDTO object should ensure that null is never passed as a parameter here. But an additional nullcheck wouldn't hurt. Adding.